src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -8,17 +8,17 @@ import { BackpackMouse, BackpackMouseType } from "../../../backpack-mouse";
 
 const dataColors = {
   white: {
-    mice: "#f4ce83",
+    mice: "#dfc39d",
     environment: "rgb(251,235,205)",
     environmentHighlight: "rgba(251,235,205,.7)"
   },
   neutral: {
-    mice: "#db9e26",
+    mice: "#bf8f53",
     environment: "rgb(241,216,168)",
     environmentHighlight: "rgba(241,216,168, .7)"
   },
   brown: {
-    mice: "#795423",
+    mice: "#5d3515",
```

Comment:
Did we have these colors wrong?

Reply:
This controls the line color of the populations graph. I think Michael went back on forth on it a few times, but these are the colors from https://hy7r7v.axshare.com/#id=8tunqt&p=connectedbio_-_specs__v2_7_3_ 

It's actually already in our css as `color-data-mousebrown-dark-rep`, and used in several other places such as the breeding pie charts.

Reply:
Good catch!

Reply:
Seems like a place where importing variables from SASS might make sense.